### DISCUSSION TITLE
how to specific the template directory

### DISCUSSION THREAD

**Discussion by Caratpine** [projectdiscovery/nuclei#4023] [Q&A]
nuclei version 2.9.10
my templates directory is pocs/nuclei, and I execute nuclei:
```
nuclei -u {url} -t ./poc/nuclei/
```
it doesn't seem to take effect, it automatically installs nuclei-templates for me and executes

**Comment by MetzinAround** [comment]
If you want to change the directory you want your templates to download to, you can do that in the config file. Should be in HOME/.config/nuclei. 
```
# custom directory to install / update nuclei-templates
#update-template-dir:
```


If you want to avoid downloading default templates entirely and only run yours, this command should work. 

`nuclei -u {url} -t ./poc/nuclei/ -duc`

thanks @ehsandeep for the assist.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
